handler/src/main/java/io/netty/handler/ssl/SslHandler.java

```diff
@@ -538,18 +548,14 @@ public void flush(ChannelHandlerContext ctx) throws Exception {
         }
         try {
             wrap(ctx, false);
-        } catch (Throwable cause) {
-            // Fail pending writes.
-            pendingUnencryptedWrites.removeAndFailAll(cause);
```

Comment:
Exactly, but consider that part of the code might be in the application, if Netty swallows the OOME the application won't have the opportunity to handle it.